jupyterlab_git/git.py

```diff
@@ -2183,6 +2183,26 @@ async def apply_stash(self, path: str, stash_index: Optional[int] = None) -> dic
 
         return {"code": code, "message": output.strip()}
 
+    async def submodule(self, path):
+        """
+        Execute git submodule status --recursive
+        """
+
+        cmd = ["git", "submodule", "status", "--recursive"]
+
+        code, output, error = await self.__execute(cmd, cwd=path)
+
+        results = []
+        for sha, name, commit in (
```

Comment:
Trying locally with https://github.com/denoland/deno, this seems to be giving the following error:

```
File "/home/jtp/tmp/jupyterlab_git-releaser-dist-321/.pixi/envs/default/lib/python3.13/site-packages/jupyterlab_git/git.py", line 2196, in submodule
        for sha, name, commit in (
            ^^^^^^^^^^^^^^^^^
    ValueError: not enough values to unpack (expected 3, got 2)
```

Which could be because the `git submodule status --recursive` command returns only two elements per line?

```git submodule status --recursive
-af4c6a1eee825f19d3b3cce74cfdd03ebe1a3b92 cli/bench/testdata/lsp_benchdata
-d12a68fc4930062c0bed26447a6b5245697e77c1 tests/node_compat/runner/suite
-c5d7930d5700dbbbedea607f9cf3e50acdc33e2d tests/util/std
-e78446e34a1921371658a5df08c71d83f50a2a2f tests/wpt/suite
```

Reply:
So it looks like the output changes once `git submodule update` has been ran.

Output before running `git submodule update`:
```
-af4c6a1eee825f19d3b3cce74cfdd03ebe1a3b92 cli/bench/testdata/lsp_benchdata
-d12a68fc4930062c0bed26447a6b5245697e77c1 tests/node_compat/runner/suite
-c5d7930d5700dbbbedea607f9cf3e50acdc33e2d tests/util/std
-e78446e34a1921371658a5df08c71d83f50a2a2f tests/wpt/suite
```

output after:
```
 af4c6a1eee825f19d3b3cce74cfdd03ebe1a3b92 cli/bench/testdata/lsp_benchdata (af4c6a1e)
 d12a68fc4930062c0bed26447a6b5245697e77c1 tests/node_compat/runner/suite (heads/main)
 c5d7930d5700dbbbedea607f9cf3e50acdc33e2d tests/util/std (c5d7930d)
 e78446e34a1921371658a5df08c71d83f50a2a2f tests/wpt/suite (e78446e34a)
```

I just pushed a quick fix.